index.bs

```diff
@@ -1968,53 +1972,90 @@ Constructors</h4>
+                    In case of failure, abort the following steps.
 
-                1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}} to "{{AudioContextState/running}}".
+                1. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
 
                 1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
-                    queue a media element task</a> to <a spec="dom" lt="fire an event">fire an event
-                    </a> named {{BaseAudioContext/statechange}} at the {{AudioContext}}.
-        </del>
-        <ins cite=#2400>
-            1. Attempt to <a href="#acquiring">acquire system resources</a> to use a
-                following audio output device based on {{AudioContext/[[sink ID]]}} for
-                rendering:
+                    queue a media element task</a> to execute the following steps:
 
-                * The default audio output device for the empty string.
-                * A audio output device identified by {{AudioContext/[[sink ID]]}}.
-            
-                In case of failure, abort the following steps.
+                    1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}} to "{{AudioContextState/running}}".
 
-            1. Set the {{[[rendering thread state]]}} to <code>running</code> on the
-                {{AudioContext}}.
+                    1. [=Queue a media element task=] to [=fire an event=] named
+                        {{BaseAudioContext/statechange}} at the {{AudioContext}}.
 
-            1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
-                Queue a media element task</a> to execute the following steps:
+            </del>
+            <ins cite="#2400 #2567">
+                1. Let |document| be the [=current settings object=]'s [=relevant global object=]'s
+                        [=associated Document=].
 
-                1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}}
-                    to "{{AudioContextState/running}}".
+                1. Attempt to [=acquire system resources=] to use a following audio output device
+                    based on {{AudioContext/[[sink ID]]}} for rendering:
 
-                1. <a spec="dom" lt="fire an event">Fire an event</a> named
-                    {{BaseAudioContext/statechange}} at the {{AudioContext}}.
-        </ins>
+                    * The default audio output device for the empty string.
+                    * A audio output device identified by {{AudioContext/[[sink ID]]}}.
+                
+                    1. If resource acquisition fails, execute the following steps:
+                        
+                        1. If |document| is not allowed to use the feature identified by
+                            <code>"speaker-selection"</code>, abort these substeps.
```

Comment:
@padenot Please take a look at this step.

Per Chromium's privacy discussion, this step is necessary to avoid brute-forcing UUID in a drive-by fashion. If this looks good to you, I'll merge this PR.

Reply:
Sure thing, looks good.